pep-0654.rst

```diff
@@ -113,25 +113,25 @@ for this purpose, in a backwards-compatible manner, and found that it is not.
 See the `Rejected Ideas`_ section for more on this.
 
 The purpose of this PEP, then, is to add the ``except*`` syntax for handling
-``ExceptionGroups`` in the interpreter, which in turn requires that
-``ExceptionGroup`` is added as a builtin type. The semantics of handling
-``ExceptionGroups`` are not backwards compatible with the current exception
+exception groups in the interpreter, which in turn requires that
+``ExceptionGroup`` is added as a builtin type. The semantics of ``except*``
+are not backwards compatible with the current exception
 handling semantics, so we are not proposing to modify the behavior of the
```

Comment:
While we're here, I would avoid the phrase "not backwards compatible" (since it's a red flag):
```suggestion
``ExceptionGroup`` is added as a builtin type. The desired semantics of
``except*`` are sufficiently different from the current exception
handling semantics that we are not proposing to modify the behavior of the
```